ar-setup.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+artifact_registry_location=us-central1
+project=$(gcloud config get-value core/project)
+
```

Comment:
Do we run the script before or after `appctl init`?

If before, the repo URL won't exist when this is executed. 
If after, the CI pipeline won't work before this is executed, so the code/configs of the app repo shouldn't be pushed before this script.

Reply:
This script is supposed to run after `apptl init`.

>  the code/configs of the app repo shouldn't be pushed before this script.
I agree. We should add the step of running this script into the user guide.